test/integration/rules/aria-allowed-attr/failures.html

```diff
@@ -20,4 +20,5 @@
 <div role="subscript" aria-labelledby="value" id="fail20">fail</div>
 <div role="superscript" aria-label="value" id="fail21">fail</div>
 <div role="superscript" aria-labelledby="value" id="fail22">fail</div>
+<fieldset aria-required="false" id="fail23">fail</fieldset>
```

Comment:
I think we're a little too hasty with this. This particular example will definitely not cause accessibility issues. aria-required=false does nothing. Even with aria-required=true... if there is some other indication that the element is required, I don't think we can call that out as a failure of WCAG TBH.

I feel like I should have spotted that a little sooner, but I think this is something that needs review at best, and I would argue only if it's aria-required=true, and the form controls in the fieldset don't have required or aria-required.